extension-manifest-v3/src/background/contextual-conboarding.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Ghostery Browser Extension
+ * https://www.ghostery.com/
+ *
+ * Copyright 2017-present Ghostery GmbH. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+
+function openIframe(tabId) {
```

Comment:
I would use `src/background/onboarding.js`, and add this there. It is the same feature, but only another representation - no need to a new file here.